.github/ISSUE_TEMPLATE/rollout.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Rollout Request
+about: Create an issue to track the rollout of a new release
+title: ''
+labels: rollout
+assignees: ''
+
+---
+
+<!-- Please answer these questions when reporting a bug. Thanks! -->
+
+- [ ] Cut release v1.x.y-rc0 - ETA: 06/28/2021
+
+- [ ] Rollout v1.x.y-rc0 to testnet - ETA: 06/28/2021
+
+- [ ] Stress test v1.x.y-rc0 on testnet - ETA: 07/05/2021
+
+- [ ] Rollout v1.x.y-rc0 to canary API nodes on mainnet - ETA: 07/12/2021
+
+- [ ] Rollout v1.x.y-rc0 to canary delegate nodes on mainnet - ETA: 07/14/2021
```

Comment:
i think these 2 need to be after Hit critical problems (it could potentially be a hard-fork for a critical problem)?

we rollout on mainnet, ONLY after fix is merged, and we done testing v1.x.y-rcN+1, and promote to formal v1.x.y

Reply:
we can roll out to the canary API nodes on the mainnet with a relatively big hard fork height. I removed the `roll out to canary delegate node on mainnet`.